platform/security/security-services-app/src/main/resources/etc/Claims_Handler_Manager-ddf.security.sts.claimsHandler.ClaimsHandlerManager.config

```diff
@@ -1,15 +1,15 @@
-groupBaseDn="ou\=groups,dc\=example,dc\=com"
-ldapBindUserDn="cn\=admin"
-memberNameAttribute="member"
-objectClass="groupOfNames"
-password="secret"
-propertyFileLocation="etc/ws-security/attributeMap.properties"
-startTls=B"false"
-url="ldaps://${org.codice.ddf.system.hostname}:1636"
-userBaseDn="ou\=users,dc\=example,dc\=com"
-membershipUserAttribute="uid"
-loginUserAttribute="uid"
-realm=""
-kdcAddress=""
-bindMethod="Simple"
-overrideCertDn=B"false"
\ No newline at end of file
+groupBaseDn = "ou\ = groups,dc\ = example,dc\ = com"
```

Comment:
❗️We may want to avoid spaces inside DNs to avoid any problems with different versions of LDAP and AD.